Add link to support page (and indirectly to the support portal) (#15167)
digital-asset/daml: docs/source/support/support.rst

  We can't answer questions in private messages or over email, so please only ask questions in public forums.
- **Level of support**: We're happy to answer questions about error messages you're encountering, or discuss Daml design questions. However, we can't provide more extensive consultation on how to build your Daml application or the languages, frameworks, libraries and tools you may use to build it.

> If you need private support, Digital Asset offers paid support, which can be accessed `here <https://www.digitalasset.com/contact-support>`_. Digital Asset further offers consultation on how to build your Daml application. Please contact Digital Asset to ask about pricing.

Reviewer: ```suggestion
Digital Asset offers paid private support, which can be accessed `here <https://www.digitalasset.com/contact-support>`_. Digital Asset further offers consultation on how to build your Daml application; please contact us for pricing.
```
